test/FastExpressionCompiler.TestsRunner/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace FastExpressionCompiler.UnitTests
+{
+    
+    public class Program
+    {
+        public static void Main()
+        {
+            // TODO: Use NUnit classes to run tests. But better to migrate to xunit (xunit is de facto standard fot testing ALL MS stuff)
```

Comment:
Thanks for the project. Simple thing to debug the test. The same approach is taken by `Expecto`. Interesting, will it work as NUnit runner.